src/responder/nss/nsssrv_mmap_cache.c

```diff
@@ -722,6 +722,51 @@ static errno_t sss_mmap_cache_invalidate(struct sss_mc_ctx *mcc,
     return EOK;
 }
 
+static errno_t sss_mmap_cache_validate_or_reinit(struct sss_mc_ctx **_mcc)
+{
+    struct sss_mc_ctx *mcc = *_mcc;
+    struct stat fdstat;
+    bool reinit = false;
+    errno_t ret;
+
+    /* No mcc initialized? That's bug in the code. */
```

Comment:
Not necessarily a "bug".
It can be disabled, for example, by setting `memcache_timeout` or `memcache_size_*` to 0.

Reply:
Nice catch. I'll test it and update the code.

Reply:
I didn't hit the code (mcc == NULL is hit earlier in the code), but I removed the debug message anyway. It wasn't there before, so lets keep it this way.